x-pack/plugins/ml/tsconfig.json

```diff
@@ -1,3 +0,0 @@
-{
-  "extends": "../../tsconfig.json"
-}
```

Comment:
Not sure from the diff if the file is now empty or was removed?

Reply:
removed